config/samples/dataplane_v1beta1_openstackdataplane_baremetal_with_ipam.yaml

```diff
@@ -1,103 +1,80 @@
 ---
 apiVersion: dataplane.openstack.org/v1beta1
-kind: OpenStackDataPlane
+kind: OpenStackDataPlaneNodeSet
```

Comment:
This sample file just had everything I needed for the example. But yeah, I can change the name.

Reply:
I think we have general agreement on at least the API design aspect of this proposal, so I'll go ahead and update the remaining samples and change filenames to be more representative of the resources they create.

I think we just need to agree on what the `OpenStackDataPlane` CR will look like. We're still a bit stuck on how it will work based on other conversations here. So I'll just leave those ones for now and update anything that was role to nodeSet based on the new design.